packit_service/models.py

```diff
@@ -440,6 +442,27 @@ def get_all(cls) -> Optional[Iterable["CoprBuildModel"]]:
         with get_sa_session() as session:
             return session.query(CoprBuildModel).order_by(desc(CoprBuildModel.id)).all()
 
+    @classmethod
+    def get_merged_chroots(
+        cls, first: int, last: int
+    ) -> Optional[Iterable["CoprBuildModel"]]:
+        with get_sa_session() as session:
+            builds_list = (
+                session.query(
+                    # Merged row will keep the id of any one row
+                    func.min(CoprBuildModel.id).label("new_id"),
+                    # Add all the fields which MUST be the same (repeat in group_by as well)
```

Comment:
Is this comment still relevant?

Reply:
No, I'll replace it with something more suitable